registration/tests/models.py

```diff
@@ -424,3 +424,594 @@ def test_resend_activation_email_expired_user(self):
             site=Site.objects.get_current(),
         ))
         self.assertEqual(len(mail.outbox), 0)
+
+
+class SupervisedRegistrationModelTests(TestCase):
```

Comment:
same here, just add the tests for your new functionality and let the super class test the other methods